scene/animation/animation_blend_space_1d.h

```diff
@@ -63,6 +63,8 @@ class AnimationNodeBlendSpace1D : public AnimationRootNode {
 	StringName blend_position = "blend_position";
 
 protected:
+	bool sync = false;
```

Comment:
Would it make sense to enable sync by default for BlendSpace1D/2D? I imagine it would be needed for the most common use cases anyway.
```suggestion
	bool sync = true;
```

Reply:
It would make sense for it to default to true at least for BlendSpace2D, but I don't know about BlendSpace1D. Or maybe it should default to true for all syncs for consistency.
@reduz How do you think?

Reply:
For BlendSpace1D, I'm thinking about use cases like blending between walking and running without any strafing movement. Where velocity is the only input that you need.

Reply:
That means it is the same thing as NodeBlend2 and NodeBlend3, so if we set the default for BlendSpace1D to true, then the other Nodes should also set sync = true.